ujenkins/endpoints/builds.py

```diff
@@ -35,20 +64,36 @@ def get(self, name: str) -> List[dict]:
             ]
 
         Args:
-            name (str):
-                Job name or path (if in folder).
+            name (str): Job name or path (if in folder).
```

Comment:
please look on args from different functions, and use same style (blank line between arguments) etc